scripts/interface/lint.mjs

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env zx
+import 'zx/globals';
+import {
+  cliArguments,
+  getToolchainArgument,
+  popArgument,
+  workingDirectory,
+} from '../utils.mjs';
+
+// Configure additional arguments here, e.g.:
+// ['--arg1', '--arg2', ...cliArguments()]
+const lintArgs = ['--features', 'frozen-abi', ...cliArguments()];
```

Comment:
Should the linting use `--all-features` and `--all-targets`?

Reply:
Yep, good call.